torch/optim/radam.py

```diff
@@ -44,6 +44,10 @@ class RAdam(Optimizer):
 
     For further details regarding the algorithm we refer to `On the variance of the adaptive learning rate and beyond`_.
 
+    This implementation uses the same weight_decay implementation as Adam (were the weight_decay is applied
+    to the gradient) and not the one from AdamW (were weight_decay is applied to the update). Note that, for
+    historical reasons, this is different from the `original author implementation`_.
```

Comment:
"author's implementation"?

Reply:
Ho yes I had that before and then the link didn't work and I wasn't sure if it was because of the `'` or not. I should be able to add it back.